keel-api/keel-api.gradle.kts

```diff
@@ -1,7 +1,17 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
 plugins {
   `java-library`
 }
 
+tasks.withType<KotlinCompile> {
+  kotlinOptions {
+    languageVersion = "1.3"
+    jvmTarget = "1.8"
+    freeCompilerArgs += "-Xjvm-default=enable"
+  }
+}
```

Comment:
This can go in the root `build.gradle.kts` we already have configuration for `KotlinCompile` tasks there.

Reply:
I wasn't sure we wanted to apply that to all modules, but will do!